src/test/kotlin/uk/gov/justice/hmpps/offendersearch/services/MatchServiceTest.kt

```diff
@@ -3,7 +3,9 @@ package uk.gov.justice.hmpps.offendersearch.services
 import com.fasterxml.jackson.databind.ObjectMapper
```

Comment:
Quite big refactor, so need to view side by side.
Essentially test each of the match "cycles" in individual nested test